Dockerfile

```diff
@@ -249,14 +249,17 @@ FROM rootfs-base AS rootfs-squashfs
 COPY --from=rootfs / /rootfs
 RUN mksquashfs /rootfs /rootfs.sqsh -all-root -noappend -comp xz -Xdict-size 100% -no-progress
 
+FROM scratch AS squashfs
+COPY --from=rootfs-squashfs /rootfs.sqsh /
+
 FROM scratch AS rootfs
 COPY --from=rootfs-base /rootfs /
 
 # The initramfs target provides the Talos initramfs image.
 
 FROM build AS initramfs-archive
 WORKDIR /initramfs
-COPY --from=rootfs-squashfs /rootfs.sqsh .
+COPY --from=squashfs /rootfs.sqsh .
```

Comment:
These changes are necessarily required, I just found them useful when stepping through this. It gives users the options at least to just generate the squashfs.